integration_tests/run-tests.sh

```diff
@@ -19,6 +19,12 @@ TESTS_EXIT_CODE=$?
 
 cp reports/testReport.xml $TEST_REPORT_ABSOLUTE_DIR/junit.xml
 
-if [ $TESTS_EXIT_CODE -ne 0 ]; then
+npm run test:ui
```

Comment:
You might want to abort before this if the int-tests fail. But that's ultimately for you to decide if you think it's worthwhile running the ui-tests with failing int-tests.

Reply:
Its worth to run the ui-tests even with failing int-tests as required UI data is directly uploaded to btm_extract_data folder